examples/models/phi-3-mini/sentence_piece_tokenizer.h

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright (c) Meta Platforms, Inc. and affiliates.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+#include <sstream>
+
+#include <sentencepiece_processor.h>
+
+class SentencePieceTokenizer {
```

Comment:
Ideally this should live in the tokenizer folder. I understand that it is under llama2 now so not suitable for reuse/extend.

Reply:
Any chance you can add a test to this? Feel free to upload a small artifact for testing purpose.

Reply:
Same here, please add a TODO

Reply:
@larryliu0820 can we migrate our tokenizers into executorch/extensions/llm/tokenizers/...?

Reply:
I think @helunwencser is doing that